WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -44,6 +44,7 @@ open class QuickStartTourGuide: NSObject {
 
     @objc func remove(from blog: Blog) {
         blog.removeAllTours()
+        NotificationCenter.default.post(name: .QuickStartTourElementChangedNotification, object: self)
```

Comment:
What do you think about changing `BlogDashboardViewController.showQuickStart` to `toggleQuickStart`? Or `showOrHideQuickStart`?

Reply:
Agree! Fixed in 10026ed3a317a3c6124425e934f73b8e7253019c